File: app/src/main/java/com/erencol/sermon/Data/Service/IReligious.java
package com.erencol.sermon.Data.Service;

import com.erencol.sermon.model.ReligiousDays;
import com.erencol.sermon.model.Sermon;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface IReligious {
    @GET Observable<ReligiousDays> getReligiousDays(@Url String url);
}

File: app/src/main/java/com/erencol/sermon/Data/Service/IReligious.kt
package com.erencol.sermon.data.service

import com.erencol.sermon.model.ReligiousDays
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Url

interface IReligious {
    @GET
    fun getReligiousDays(@Url url: String): Observable<ReligiousDays>
}
